piet/src/lib.rs

```diff
@@ -4,13 +4,17 @@
 //! graphics API, in the tradition of [`PostScript`]. It is built on top of
 //! [`kurbo`], a 2D geometry library.
 //!
-//! The main interface is the [`RenderContext`] trait.
+//! The main interface is the [`RenderContext`] trait. Another useful place
+//! to look at if you're getting started would be the `Device` and
+//! `BitmapTarget` documentation on [`piet-common`], to create `RenderContext`
+//! objects and extract rendered pixels from them.
 //!
 //! This API can be implemented on various platforms, allowing drawing code
 //! to be reused in an approximately consistent way. Various such implementations
 //! exist, such as [`piet-cairo`], [`piet-coregraphics`], and [`piet-direct2d`].
```

Comment:
forgive my micro-management, but I would prefer to have the section mentioning piet-common be at the bottom, after we mention the other crates:

```suggestion
//! exist, such as [`piet-cairo`], [`piet-coregraphics`], and [`piet-direct2d`].
//!
//! If you are interested in using piet to target multiple platforms,
//! [`piet-common`] is a utility crate that re-exports an appropriate
//! backend based on the compile target, and provides convenience
//! types and methods for setting up a [`RenderContext`].
```